src/cli/vale-ai-command.ts

```diff
@@ -0,0 +1,140 @@
+import type { Command } from 'commander';
+import { ValeRunner } from '../evaluators/vale-ai/vale-runner';
+import { ValeAIEvaluator } from '../evaluators/vale-ai/vale-ai-evaluator';
+import { createProvider } from '../providers/provider-factory';
+import { DefaultRequestBuilder } from '../providers/request-builder';
+import { loadDirective } from '../prompts/directive-loader';
+import { parseCliOptions, parseEnvironment } from '../boundaries/index';
+import { handleUnknownError } from '../errors/index';
+import type { ValeAIConfig } from '../evaluators/vale-ai/types';
+
+/*
+ * Registers the 'vale-ai' command with Commander.
+ * This command runs Vale CLI with AI-enhanced suggestions for writing improvements.
+ */
+export function registerValeAICommand(program: Command): void {
+  program
+    .command('vale-ai [files...]')
+    .description('Run Vale with AI-enhanced suggestions')
+    .option('-v, --verbose', 'Enable verbose logging')
+    .option('--context-window <size>', 'Number of characters to extract before/after each issue for AI context', '100')
```

Comment:
Since it's now an evaluator system, and every evaluator would need to register its command, this should be made a method on the evaluator class, so every evaluator defines its Command registration method. 

We could do that on a separate PR to keep things clean.